CONTRIBUTING.rst

```diff
@@ -37,3 +37,21 @@ Look in ``.github/workflows/`` to find the commands.
 For example, to run tests::
 
     $ python3 -m pytest
+
+
+Making a release
+----------------
+
+``flake8-pyi`` uses calendar-based versioning. For example, the first
+release in January 2022 should be called 22.1.0, followed by 22.1.1.
+
+Releasing a new version is easy:
+
+- Make a PR that updates the version header in ``CHANGELOG.rst``
+  and the ``__version__`` attribute in ``pyi.py``.
+- Merge the PR and wait for tests to complete.
+- Draft a release using the GitHub UI. The tag name should be
+  identical to the version (e.g., ``22.1.0``) and the release notes
+  should be copied from ``CHANGELOG.rst``.
```

Comment:
I have automated this in one of my projects with a mess of `awk` to extract the relevant part of changelog, and `actions/create-release` to make the release:  https://github.com/Akuli/porcupine/blob/1ad0d5fa8d944ef66a0dbb7e0100da3a13aebfda/.github/workflows/build.yml To trigger it, I use a script that sanity checks the CHANGELOG, changes the version numbers and creates a tag.

That said, we can automate more of this later, and for now I think we just want something that works.

Reply:
I believe we should at least do a sanity check in the script to see whether the versions in the changelog, the setup file and the tag match, although we can do that in a later version as well.